src/haxelib/server/Repo.hx

```diff
@@ -80,6 +80,15 @@ class Repo implements SiteApi {
 		};
 	}
 
+	public function getLatestVersion( project : String ) : SemVer {
+		var p = Project.manager.select($name == project);
+		if( p == null )
+			throw "No such Project : "+project;
+
+		var vl = Version.manager.search($project == p.id, { orderBy: [-major, -minor, -patch, -previewNum], limit: 1});
```

Comment:
Oh right, for some reason I thought those where the same field.
Will have to check some sql, no idea on how enums work there.